helm/hpcc/values.yaml

```diff
@@ -79,6 +79,14 @@ security:
 
 ## storage:
 ##
+## 1. If a component has the storagePlane property set, then that plane will be the default data location for that component.
+## 2. If storage.dataStorage.plane is defined then that plane will be the default location
+## 3. If there is a plane definition with no label or a label of "data" then the first matching plane will be the default data location
+## 4. Otherwise the default data location will be derived from the properties of storage.dataStorage.
```

Comment:
I'm not sure 3 and 4 are helpful, I think they may complicate and confuse.
If we make it a requirement that there the global default data plane must be defined storage.dataStorage.plane, that that plane must exist. I think that will be clearer and template code simpler.

Reply:
(3) is could be avoided.  Same also applies to dali, dlls spill etc..  It avoids having to potentially specify the same thing in two different places.
(4) is needed until all examples and documentation switches over to defining a list of planes.